scripts/examples/gn_asr_example.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
```

Comment:
please integrate with build_examples.py and avoid having more examples shell scripts.